### PR TITLE
I am adding a ipynb that explores the possibilities of using a transfer learning model.

### PR DESCRIPTION
**Pull request by Dimkoim**

### CODE REVIEW

File: material/notebooks/Transfer_learning.ipynb [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data_dir = \"../data/sample_new/train\"\n",
+    "validation_data_dir = \"../data/validation\"\n",
```
Comment (san89) [inline material/notebooks/Transfer_learning.ipynb]: Mimis, it seems that this validation folder is never created from the current code that we are working with. In Kaggle, there is only train and test files, so my guess is that validation is a subset of the train set.

I suggest to first push the code that creates the validation data set and then we can proceed with this pull request.

Comment (steremma) [reply material/notebooks/Transfer_learning.ipynb]: It is the responsibility of each of us to download the dataset with the provided `Download.py` file. There is even an example in the `Download.ipynb` notebook for the train and test sets. Just add two lines to get the validation set and feel free to push this in a separate PR if you feel other collaborators may find it useful

**Comment by steremma** [comment]
I agree with Santiago's second point, especially since these models take so long to train. About the validation set, I would expect everyone to either have the dataset download already or at least be able to do it using the notebook (its 2 lines of code I think).
